hyperspy/model.py

```diff
@@ -1722,12 +1722,12 @@ def __getitem__(self, slices):
             slices,
             self.isNavigation)
         _signal = self.model.signal._slicer(slices, self.isNavigation)
-        if _signal.metadata.Signal.signal_type == 'EELS':
-            _model = _signal.create_model(
-                auto_background=False,
-                auto_add_edges=False)
-        else:
-            _model = _signal.create_model()
+        # let's not do anything automatically, since then it's difficult to keep
```

Comment:
I would rather move this to `next_minor` and at the same time remove autoadding anything. As you can see it used to be checking if it's an EELS model, etc., but then EDS model was added and hence the bug. In general I don't like adding exceptions where either a general approach or sane defaults should suffice...

In this case the "sane defaults" might include "sane kwarg names", making the current approach passable ;)